app/models/agents/website_agent.rb

```diff
@@ -166,6 +169,25 @@ def validate_options
       validate_web_request_options!
     end
 
+    def validate_consider_http_success_option!
+      consider_success = options["consider_http_error_success"]
+      if consider_success != nil
```

Comment:
I think it'd be easiest on the user if a string or integer were both fine like that, yes.